dev/bender/plugins/ckeditor5/static/extensions.js

```diff
@@ -0,0 +1,90 @@
+/**
+ * @license Copyright (c) 2003-2015, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/* globals bender, CKEDITOR, describe: true */
+
+'use strict';
+
+// ### General extensions.
+( function() {
+	// Make Bender wait to start running tests.
+	var done = bender.defer();
+
+	// Wait for the "ckeditor" module to be ready to start testing.
+	CKEDITOR.require( [ 'ckeditor' ], done );
```

Comment:
It may be needed if `bender.amd.require()` is not used (e.g., the amd test). I think it is better to keep it just in case and it should not be a big deal anyway.